Clic&Farm-iOS/Controllers/AddInterventionViewController.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  AddInterventionViewController.swift
+//  Clic&Farm-iOS
+//
+//  Created by Guillaume Roux on 30/07/2018.
+//  Copyright © 2018 Ekylibre. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class AddInterventionViewController: UIViewController {
+    
+    //MARK : Properties
+    
+    @IBOutlet weak var dimView: UIView!
+    @IBOutlet weak var selectCropsView: UIView!
+    @IBOutlet weak var interventionToolsTableView: UITableView!
+    @IBOutlet weak var validateButton: UIButton!
+    @IBOutlet weak var navigationBar: UINavigationBar!
+    @IBOutlet weak var heightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var firstView: UIView!
+    @IBOutlet weak var collapseButton: UIButton!
+    
+    var interventionTools = [NSManagedObject]() {
+        didSet {
+            if interventionTools.count > 0 {
+                interventionToolsTableView.reloadData()
+            } else {
+                interventionToolsTableView.isHidden = true
+            }
+        }
+    }
+    var interventionType: String?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        UIApplication.shared.statusBarView?.backgroundColor = UIColor(rgb: 0x175FC8)
+        
+        // Adds type label on the navigation bar
+        let navigationItem = UINavigationItem(title: "")
+        let typeLabel = UILabel()
+        if interventionType != nil {
+            typeLabel.text = interventionType
+        }
+        typeLabel.textColor = UIColor.white
+        typeLabel.sizeToFit()
+        
+        let leftItem = UIBarButtonItem.init(customView: typeLabel)
+        navigationItem.leftBarButtonItem = leftItem
+        navigationBar.setItems([navigationItem], animated: false)
+        
+        selectCropsView.clipsToBounds = true
+        selectCropsView.layer.cornerRadius = 3
+        
+        interventionToolsTableView.layer.borderWidth  = 0.5
+        interventionToolsTableView.layer.borderColor = UIColor.lightGray.cgColor
+        interventionToolsTableView.layer.cornerRadius = 4
+        
+        // Crops select
+        validateButton.layer.cornerRadius = 3
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
```

Comment:
Why do we override `didReceiveMemoryWarning()` if we don't deallocate anything specific?